process.py

```diff
@@ -498,9 +505,8 @@ def process_arguments()->Tuple[TextIO, TextIO, bool, bool, bool]:
     max_references: int = 0
 
     INPUT_FILE, OUTPUT_FILE, DOT_FILE, PRINT_TREE, SINGLE_PARENT,\
-    IS_MISSING_FEATURE = process_arguments()
+    IS_MISSING_FEATURE,CONTENTS = process_arguments()
```

Comment:
Please be mindful of formatting. Need a space after the last comma.

Reply:
Fixed formatting